queries/buildings.jinja2

```diff
@@ -82,6 +86,9 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+{% if zoom >= 16 %}
+    16 AS min_zoom,
```

Comment:
Good call on adding `min_zoom`. Would we want this to be 17 to match the previous behavior, before z16 metatiling?